stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -0,0 +1,68 @@
+[id="proc-aap-aws-backup-and-recovery"]
+
+In order to backup and restore your AAP deployment, it is vital to ensure automatic backups have been set for EFS and that these backups are accessible for restoration. Upon creating an AAP deployment, automatic EFS backups are set by default, however it is still prudent to check if those backups can be restored before a disaster scenario. In addition to this, it is also extremely important that regular manual snapshots of the RDS are taken to ensure a deployment can be restored as close as possible to its previous working state. Follow the steps below to ensure a smooth backup and recovery process.
+
+== Backup Process
+
+=== Take a manual snapshot of the RDS for your AAP deployment
+. Navigate to *CloudFormation* on your AWS Console
+. Select the *Resources* tab and navigate to menu:rds[rds] on the resource tree and click the link to go to the RDS of the deployment
```

Comment:
. Select the *Resources* tab.
. On the resource tree, select *rds > rds* and click the link to go to the RDS of the deployment.

Is the menu item rds or RDS?

Reply:
The menu item is rds > rds
<img width="969" alt="Screenshot 2022-12-07 at 10 25 05 AM" src="https://user-images.githubusercontent.com/112966709/206219845-f74453cc-21bb-43b5-9ee4-ed12883e011e.png">